Formula/yarn.rb

```diff
@@ -17,7 +15,7 @@ class Yarn < Formula
   depends_on "node"
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    cp_r '.', prefix
```

Comment:
Did you test this locally? This is 100% not going to work because there's e.g. no `libexec`.